torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -976,6 +980,38 @@ def test_profiler_with_script_remote_rpc(self):
             use_record_function=True,
         )
 
+    @dist_init
+    def test_server_process_global_profiler(self):
```

Comment:
Should we also test what happens in the case of double calls, calling exit without enter, calling exit twice, entering twice, etc?

Reply:
Adding.